faker/providers/person/__init__.py

```diff
@@ -10,6 +10,8 @@ class Provider(BaseProvider):
 
     last_names = ['Doe', ]
 
+    middle_names = []
```

Comment:
Not every provider supports middle names. You can move this logic to the localized russian subclass (`person/ru_RU/__init__.py`)